com.unicornsonlsd.Finamp.metainfo.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.unicornsonlsd.Finamp</id>
+  <name>Finamp</name>
+  <summary>An open source Jellyfin music player</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>AGPL-3.0-only</project_license>
+  <recommends>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </recommends>
+  <developer_name>jmshrv</developer_name>
+  <url type="homepage">https://github.com/jmshrv/finamp</url>
+  <description>
+    <p>Finamp is an open source, cross-platform Jellyfin music player</p>
+    <p>It supports streaming playback and offline downloads, playlists, favourites and shuffle. Linux support is currently in Beta.</p> 
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://raw.githubusercontent.com/baarkerlounger/flathub/finamp/screenshots/main.png</image>
```

Comment:
a direct link to git should always point at a specific commit. This is not the case.